### PR TITLE
Module migration from old utils to new SDK

### PR DESCRIPTION
**Pull request by mogith-pn**
https://clarifai.atlassian.net/browse/TT-2958

### CODE REVIEW

File: pages/grid.py [hunk 1]
```diff
@@ -2,13 +2,12 @@
 
 import numpy as np
 import streamlit as st
-from clarifai.auth.helper import ClarifaiAuthHelper
-from clarifai.client import create_stub
-from clarifai.listing.lister import ClarifaiResourceLister
+from clarifai.client.auth import create_stub
```
Comment (sanjaychelliah) [inline pages/grid.py:5]: This import is not needed..

File: pages/mosaic.py [hunk 2]
```diff
@@ -1,11 +1,11 @@
 import streamlit as st
-from clarifai.auth.helper import ClarifaiAuthHelper
-from clarifai.client import create_stub
-from clarifai.listing.lister import ClarifaiResourceLister
 from clarifai.modules.css import ClarifaiStreamlitCSS
-from stqdm import stqdm
+from clarifai_grpc.grpc.api import service_pb2
+from clarifai.client.auth import create_stub
```
Comment (sanjaychelliah) [inline pages/mosaic.py:4]: Both `service_pb2` and `create_stub `  are not needed here..

File: pages/mosaic.py [hunk 3]
```diff
@@ -15,7 +15,6 @@
 stub = create_stub(auth)
 metadata = auth.metadata
```
Comment (sanjaychelliah) [inline pages/mosaic.py:16]: `metadata` variable is not used anywhere in this script.

File: utils/api_utils.py
Comment (sanjaychelliah) [inline utils/api_utils.py:1]: `predict_from_image` and `predict_from_text` can be directly used from the SDK in all the scripts..

**Review by sanjaychelliah — COMMENTED** [review]
Some comments..

File: pages/ner.py [hunk 4]
```diff
@@ -18,29 +20,29 @@
 model_id = 'ner-english'
 model_version_id = 'a813ff5b362c41f790c506b871e7dea4'
```
Comment (sanjaychelliah) [inline pages/ner.py:21]: Can we directly use something like this `model_prediction = Model("https://clarifai.com/anthropic/completion/models/claude-v2").predict_by_bytes(b"Write a tweet on future of AI", "text")`

**Review by sanjaychelliah — CHANGES_REQUESTED** [review]
Create stub import `from clarifai.client.auth import create_stub ` is not needed in any script except in `api_utils.py`

**Comment by mogith-pn** [comment]
> Create stub import `from clarifai.client.auth import create_stub ` is not needed in any script except in `api_utils.py`

Hi Sanjay,
We are using `UserDataObject` as arguments in functions inside grid.py, metrics.py, mosaic.py as well. I have removed it from compare.py and from ner.py

**Review by sanjaychelliah — APPROVED** [review]
LGTM.

**Pull request merged**